README.md

```diff
@@ -78,6 +78,17 @@ Note that Next has some special behaviour when it comes to passing environment v
 | `NEXT_PUBLIC_MATOMO_ENABLED`                  | Control whether Matomo is enabled.                                                                                                                                                                                                                         |
 | `NEXT_PUBLIC_MATOMO_SITE_ID`                  | If Matomo is in use, provide site id to direct analytics into the correct destination.                                                                                                                                                                     |
 | `NEXT_PUBLIC_DEFAULT_ISR_REVALIDATE_SECONDS`  | Default length of time in seconds between [NextJS's ISR's](https://nextjs.org/docs/basic-features/data-fetching/incremental-static-regeneration) regenerations. _**Default:** `10`_                                                                        |
+| `NEXT_PUBLIC_SENTRY_ENVIRONMENT`              | Environment used for sentry logging                                                                                                                                                                                                                        |
+| `NEXT_PUBLIC_SENTRY_DSN`                      | Sentry Data Source Name (DSN) [DSN Docs](https://docs.sentry.io/product/sentry-basics/dsn-explainer/)                                                                                                                                                      |
```

Comment:
These variables have been described earlier on in the document. These new descriptions are better, so the old ones should be removed.

Looks good otherwise 👍

Reply:
good point :D